source/includes/_chat_summary.md

```diff
@@ -378,3 +542,20 @@ https://api.reachlocalservices.com/client_reports/chat_summary/TEST_1?start_date
 |chat_service|Integer|Number of chats marked as 'service'|
 |chat_other|Integer|Number of chats marked as 'other'|
 |call_connect|Integer|Number of calls connected|
+
+<internal>
```

Comment:
Unfortunately we have to tag each element that get's rendered into html.  So tag, `<internal>**Web Publisher Campaign Object**</internal>` and tag the table with `|<internal>Field Name|Datatype|Description</internal>|`

Reply:
Fixed